src/test/java/org/jpeek/AppTest.java

```diff
@@ -56,19 +57,21 @@ public void createsXmlReports() throws IOException {
         final Path output = Files.createTempDirectory("").resolve("x1");
         final Path input = Paths.get(".");
         new App(input, output).analyze();
-        MatcherAssert.assertThat(
-            Files.exists(output.resolve("LCOM.xml")),
-            Matchers.equalTo(true)
-        );
-        MatcherAssert.assertThat(
-            XSLDocument
+        new Assertion<>(
+            "LCOM.xml file exists",
+            () -> Files.exists(output.resolve("LCOM.xml")),
+            new IsTrue()
+        ).affirm();
+        new Assertion<>(
+            "Has body",
```

Comment:
@GnusinPavel same above and what's is "has body"? Please, use a more meaningful error message